moped-editor/src/views/projects/projectView/NoteInput.js

```diff
@@ -0,0 +1,236 @@
+import React, { useEffect } from "react";
+import {
+  Box,
+  Button,
+  Container,
+  Grid,
+  FormControl,
+  Radio,
+  RadioGroup,
+  FormControlLabel,
+} from "@mui/material";
+import makeStyles from "@mui/styles/makeStyles";
+import ProjectSaveButton from "../newProjectView/ProjectSaveButton";
+
+import { AutoFocusPlugin } from '@lexical/react/LexicalAutoFocusPlugin';
+import { LexicalComposer } from '@lexical/react/LexicalComposer';
+import { RichTextPlugin } from '@lexical/react/LexicalRichTextPlugin';
+import { ContentEditable } from '@lexical/react/LexicalContentEditable';
+import { HistoryPlugin } from '@lexical/react/LexicalHistoryPlugin';
+import { LexicalErrorBoundary } from '@lexical/react/LexicalErrorBoundary';
+import { useLexicalComposerContext } from '@lexical/react/LexicalComposerContext';
+import { $generateHtmlFromNodes, $generateNodesFromDOM } from '@lexical/html';
+import { $getRoot } from 'lexical';
+import { LinkPlugin } from '@lexical/react/LexicalLinkPlugin';
+import { LinkNode } from '@lexical/link';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+    backgroundColor: theme.palette.background.paper,
+  },
+  cancelButton: {
+    margin: theme.spacing(1),
+    position: "relative",
+    color: theme.palette.secondary.dark,
+  },
+  contentEditable: {
```

Comment:
This looks like a good place to add some styles to make this look more like MUI. Could you add the following? Not overly concerned about going full-blown MUI since the old Quill component had its own theme.
- border radius 4px
- border color the same as the primary text color
- font family to `theme.typography.fontFamily`

Also, could you remove the 300px height set here? The current production editor starts at one line of height and grows as needed. I think that we should keep that behavior to preserve vertical space unless we hear feedback that says otherwise. 🙏

Reply:
I also noticed a regression that we fixed before where the font color of an existing comment in edit mode is the lighter secondary color instead of the primary text color. Looks like we need to explicitly set the color of the font here to be the theme primary text color. I think we determined that it is inheriting the secondary color from its wrapper? Could you add a comment to help us next time around? Thanks!

<img width="407" alt="Screenshot 2024-08-09 at 10 26 31 AM" src="https://github.com/user-attachments/assets/19353004-67e3-4312-9902-d7c665617be3">

Reply:
updated all of these, thank you for the guidance!! it looks much nicer ✨